DigitalPassaport/DigitalPassaport/DataManagment/CoreDataManagment/CoreDataManager.swift

```diff
@@ -86,15 +70,19 @@ class CoreDataManager {
         }
     }
     
-    func fetchCredentials() -> AnyPublisher<[CredentialEntity], APIError> {
+    func saveCredentialFromResponse(_ credentialResponse: Credential, for selectedPass: Pass) -> AnyPublisher<Void, APIError> {
```

Comment:
A parameter is better to be an entity instead of a response. In that way you can't forget to save as entity instead, you force devs to use it in that way.